enclave-tls/src/tls_wrappers/openssl/un_negotiate.c

```diff
@@ -191,7 +190,7 @@ static int find_extension_from_cert(X509 *cert, const char *oid, uint8_t *data,
 		}
 
 		/* If found then get the data */
-		if (!strcmp(oid_buf, oid)) {
+		if (!strcmp((const char *)oid_buf, oid)) {
```

Comment:
Why changing to signed char?

Reply:
strcmp(const char *s1,const char *s2); the cast is (**const** char *)